src/command_modules/azure-cli-servicefabric/azure/cli/command_modules/servicefabric/commands.py

```diff
@@ -0,0 +1,49 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+
+
+from azure.cli.core.commands import cli_command
+from azure.cli.core.util import empty_on_404
+
+from ._client_factory import servicefabric_client_factory
+
+custom_path = 'azure.cli.command_modules.servicefabric.custom#{}'
+mgmt_path = 'azure.mgmt.servicefabric.operations.clusters_operations#{}'
+
+factory = lambda args: servicefabric_client_factory(**args).clusters
+
+cli_command(__name__, 'sf cluster show', mgmt_path.format('ClustersOperations.get'), factory, exception_handler=empty_on_404)
+
+cli_command(__name__, 'sf cluster list', custom_path.format('list_cluster'), factory)
+
+cli_command(__name__, 'sf cluster new', custom_path.format('new_cluster'), factory, no_wait_param=False)
+
+cli_command(__name__, 'sf cluster certificate add', custom_path.format('add_cluster_cert'), factory, no_wait_param=False)
+
+cli_command(__name__, 'sf cluster certificate remove', custom_path.format('remove_cluster_cert'), factory, no_wait_param=False)
+
+cli_command(__name__, 'sf cluster client-certificate add', custom_path.format('add_client_cert'), factory, no_wait_param=False)
+
+cli_command(__name__, 'sf cluster client-certificate remove', custom_path.format('remove_client_cert'), factory, no_wait_param=False)
+
+cli_command(__name__, 'sf cluster setting set', custom_path.format('set_cluster_setting'), factory, no_wait_param=False)
+
+cli_command(__name__, 'sf cluster setting remove', custom_path.format('remove_cluster_setting'), factory, no_wait_param=False)
+
+cli_command(__name__, 'sf cluster reliability update', custom_path.format('update_cluster_reliability_level'), factory, no_wait_param=False)
+
+cli_command(__name__, 'sf cluster durability update', custom_path.format('update_cluster_durability'), factory, no_wait_param=False)
+
+cli_command(__name__, 'sf cluster node-type add', custom_path.format('add_cluster_node_type'), factory, no_wait_param=False)
```

Comment:
this is a bug in our service, we can't have this command ready for now.  we will add when we have resolved the issue.